pkg/testing/multipass/provisioner.go

```diff
@@ -152,14 +158,9 @@ func (p *provisioner) launch(ctx context.Context, cfg runner.Config, batch runne
 		return fmt.Errorf("failed to marshal cloud-init configuration: %w", err)
 	}
 
-	p.logger.Logf("Launching multipass instance %s", batch.ID)
 	var output bytes.Buffer
-	proc, err := process.Start("multipass",
-		process.WithContext(ctx),
-		process.WithArgs(args),
-		process.WithCmdOptions(
-			runner.AttachOut(&output),
-			runner.AttachErr(&output)))
+	p.logger.Logf("Launching multipass image %s", batch.ID)
+	proc, err := process.Start("multipass", process.WithContext(ctx), process.WithArgs(args), process.WithCmdOptions(runner.AttachOut(&output), runner.AttachErr(&output)))
```

Comment:
you have multipass in a constant
```suggestion
	proc, err := process.Start(Name, process.WithContext(ctx), process.WithArgs(args), process.WithCmdOptions(runner.AttachOut(&output), runner.AttachErr(&output)))
```

Reply:
I do, but the provider name and the cli command do not need to be the same. To have the `multipass` cli command as a constant, I'd rather have another constant, which could be something like `CMDMultipass = Name`. However most, if not all, commands the framework invoke aren't on constants, so I think letting it as it's right now the best option